site/content/docs/user-guide/using-custom-indexes.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Using Custom Plugin Indexes
+slug: custom-indexes
+weight: 800
+draft: true
+---
+
+Plugin indexes contain plugin manifests, which are documents that describe
+installation procedure for a plugin. For discovery purposes, Krew comes with a
+`default` plugin index, plugins hosted in the [`krew-index`
+repository](https://github.com/kubernetes-sigs/krew-index).
+
+However, some plugin authors may choose to host their own indexes that contain
+their curation of kubectl plugins. These are called "custom plugin indexes".
+
+## Adding a custom index
```

Comment:
I think the title isn’t good. 
It would be better to explain in this section “The default index” and

* how it can be removed
* or pointed to another repo
* and how plugins without an explicit index name mean “default”

Reply:
I feel like the adding/removing/listing indexes sections are important to showcase the new index command right? I agree though that a section explaining the default index would be good, I can move the advanced usage section I had and change it to reflect the points you listed.

Reply:
Yes, I think this should work well. You could also explain how plugin names are resolved. Iow, that a `default/` prefix is added implicitly, if no index name is given. Then you could also explain how plugin name clashes are handled and remove the `Duplicate plugin names` section in the other page.